### DISCUSSION TITLE
Is it possible to let continuous text flow around an element (ie. image)?

### DISCUSSION THREAD

**Discussion by patricknick** [QuestPDF/QuestPDF#1079] [Q&A] [answered]
Hi! I would like to implement a layout as shown below in QuestPDF:

<img src="https://github.com/user-attachments/assets/86d29f60-43ce-4590-9ced-7939d0905e2a" alt="desired layout" width="300">

Example code: 
```
column.Item().Text(Placeholders.Paragraphs())
    .Justify()
    .ParagraphSpacing(8)
    .FontSize(12);

column.Item().PaddingVertical(16).Row(row =>
{
    row.RelativeItem().Text(Placeholders.Paragraph())
        .Justify()
        .ParagraphSpacing(8)
        .FontSize(12);

    row.AutoItem().Width(250).Height(150).PaddingLeft(16).Placeholder();
});

column.Item().Text(Placeholders.Paragraphs())
    .Justify()
    .ParagraphSpacing(8)
    .FontSize(12);
```

However, this requires me to manually break the text into three sections and place it accordingly. To make it perfect, the middle section would need to be exactly as long as the image, which is quite hard to calculate, I think. 
Preferably, I would like to place the image on a position fixed and relative to the page and the text to flow nicely around the object / image. Similar as it behaves in a text document, such as Word. 

Is this achievable with QuestPDF? Or does anyone know an alternative / workaround? I couldn't find any hint myself unfortunately.

**Comment by MarcinZiabek** [comment, marked as answer]
I am afraid that this particular requirement is currently not supported 😢

**Reply by patricknick** [reply]
Too bad, that would have been a very handy feature. But thank you for clarifying!👍 

P.S.: I'm currently evaluating QuestPDF for a project and enjoy it quite much. Thanks for all the work! 🙏

**Comment by DawidRobertNowak** [comment]
Hello,

We are closing this thread due to inactivity. Thank you for taking the time to reach out and for the kind words about QuestPDF.

Text wrapping around elements (similar to float behavior in CSS or Word) is not currently supported and is not on our roadmap, as we are focused on other features and improvements. We appreciate the feedback nonetheless, as community demand helps us shape future priorities.

Should you encounter any further problems, please don't hesitate to open a new issue.

Thank you!
